clang/docs/SYCLMainPageforRFC.md

```diff
@@ -0,0 +1,62 @@
+# Add full support for the SYCL programming model
+
+We (Intel) propose adding full support for the SYCL programming model support to the LLVM/Clang project to facilitate collaboration on C++ single-source heterogeneous programming for accelerators (e.g., GPU, FPGA, DSP) from different hardware and software vendors. The SYCL 2020 Specification is available at the Khronos site: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html.
+
+Our prior RFC toward this effort can be seen here: https://lists.llvm.org/pipermail/cfe-dev/2019-January/060811.html
+
+In the four years since then: 
```

Comment:
In the last four years, Here are some of the major developments in our SYCL compilation efforts: